### PR TITLE
Understanding the Resilient Packet Ring Technology

### LINKED ISSUE #5058: Understanding the Resilient Packet Ring Technology
#### Proposed title of article
Understanding the Resilient Packet Ring Technology

#### Proposed article introduction
The Resilient Packet Ring technology is MAC-Level Ethernet based that is created for metro networks. It guarantees a bigger bandwidth efficiency by allowing the sharing of links by the bandwidth. It also allows both high and low priority classes of service.

It is designed and set up to provide SDH networks resilience to increase the efficiency of the IP as well as the Ethernet services. Operations are run through ringlets set up from the creation of RPR stations for traffic flow. 

#### Key takeaways 
1. Basics of Resilient Packet Ring
2. Adopted types of transmission media.
3. Traffic queues and Class of Service
4. Limitations related to Resilient Packet Ring

#### References
N/A
### Templates to use as guides
N/A

### PR DESCRIPTION
**Pull request by eunice-wanjiku**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5058

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

File: content/articles/understanding-the-resilient-packet-ring-technology/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+### Traffic queues and Class of Service
+
+The traffic present is associated with Class of Service, which is categorized into classes A, B, and C. They signify high, medium, and low traffic simultaneously. Class A is a virtual circuit bandwidth that offers support to low latency applications such as video. Class B is a combination of the bandwidth for the virtual circuit as well as the intensity of the burst that is higher than the CIR. Class C is the most effective effort traffic that is highly flexible as it integrates any available bandwidth supporting the access traffic of the internet.
```
Comment (mbaabuones) [inline content/articles/understanding-the-resilient-packet-ring-technology/index.md:35]: How is this section connected with the topic of your article? Give further explanations to connect it with the topic of the article. If it is a sub-section, then you should use a level four header.

File: content/articles/understanding-the-resilient-packet-ring-technology/index.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+### Application of Packet Ring Technology
+
+RPR is commonly used in national SONET, metropolitan, and ethernet carrier networks. This architecture is normally incorporated into the SONET networks so that they can gain the ability to transfer packets. This is because, the technology, RPR, offers the network statistical multiplexing that is fault-tolerant. To achieve this level of fault tolerance, a resilient packet ring employs the use of two counter-rotating rings. Both of these rings are connected by fiber forming a ring topology. The nodes forming the RPR on the other hand both transmit packets in opposing directions. The two nodes, that is inner and outer rings, function simultaneously and in the event, one fails only the capacity of traffic transferred is decreased but there is no unavailability.
```
Comment (mbaabuones) [inline content/articles/understanding-the-resilient-packet-ring-technology/index.md:31]: If possible, this section can be sub-divided into sub-sections.

**Review by mbaabuones — CHANGES_REQUESTED** [review]
@eunice-wanjiku Nice article. Although the article is informative, there is room for improvement.
 - The content of the article (minus the table of contents and resources) is less than the threshold size- 750 words. Add content to reach at least 750 words. 
- I have also noticed that you have only one paragraph in every section. To make your article unique and more appealing, every section should consist of at least two paragraphs. If possible, some of the sections can be sub-divided into sub-sections. 
- There are many good articles that have described the basics of phenomena. Take time to go through them to see how they have been crafted. Here are two good examples: one on [means end analysis](https://www.section.io/engineering-education/basics-of-means-end-analysis-in-ai/) and another on [production system in AI](https://www.section.io/engineering-education/breaking-down-the-production-system-in-ai/).
- Your article has some grammatical errors. After making the suggested changes, use Grammarly.com to identify and correct grammatical errors. 
- Other changes have been suggested in the article. Ping me when you finish all the revisions.

**Comment by eunice-wanjiku** [comment]
@mbaabuones all done

**Comment by mbaabuones** [comment]
> @mbaabuones all done

@eunice-wanjiku Thank you for revising your article. However, I am still concerned that the content of your article is thin. The sections 'adopted types of transmission data', 'applications' and 'limitations' can still be expanded to add value to your article. If you decide to use sub-sections, make sure there are at least two subsections in every section. You can provide a leading statement before branching to your sub-sections. For example, in the section 'applications', you can provide a statement that highlights the main applications. After this statement, you can list them in sub-sections to give further explanations. This can also be applied in the section 'adopted types of transmission data'. Please refer to the two articles I suggested to you. Please work on these changes.

**Comment by eunice-wanjiku** [comment]
@mbaabuones Check the changes made. I believe this is what you were going for?

**Comment by mbaabuones** [comment]
> @mbaabuones Check the changes made. I believe this is what you were going for?

@eunice-wanjiku Yes. Just a minor issue. The section after 'the basics of RPT' has a level four header. Based on the new structure, this section fits to have a level three header. Change this and ensure there are at least two paragraphs in this section. There should be only one space between paragraphs. In some cases, there are two or more spaces. Remove the extra spaces.

**Comment by eunice-wanjiku** [comment]
@mbaabuones Confirm the changes

**Review by mbaabuones — APPROVED** [review]

**Comment by Kayere** [comment]
Plagiarism check done.

**Pull request merged**